src/main/java/UseCase1.java

```diff
@@ -42,9 +53,20 @@ public static void main(String[]args)
                         groupBy(customers.col("customer_id"),customers.col("customer_fname"),customers.col("customer_lname")).
                         agg(count(lit(1)).alias("customer_order_count")).orderBy(customers.col("customer_id"),col("customer_order_count").desc());
                 result.show();
-                String path=System.getenv("OUTPUT_PATH")+"\\UseCase1";
-                result.coalesce(1).write().option("header",true).mode("overwrite").csv(path);
-                logger.info("*************************File Saved Successfully***************");
+                long count=result.count();
+                if(validateResult(count))
```

Comment:
This function call will not work when we run the code in cluster mode. we have to use UDF